### DISCUSSION TITLE
Documentation to deploy on EKS or Openshift

### DISCUSSION THREAD

**Discussion by sravanakumarchakrapani** [schemaspy/schemaspy#1191] [Q&A]
Is there any documentation on how to deploy Schema spy to Openshift or EKS pod. I tried to deploy to EKS using the docker image and exposed the svc and created ingress on port 80 but I am getting bad gateway. Is there any other approach I should be hosting the output folder with different port or any other documentation for EKS

**Comment by npetzall** [comment]
SchemaSpy is a command line tool. It generates a report as a static webpage that you can deploy to http server.

I know there is a fork trying to do it as a service, but I can't remember which.

**Reply by sravanakumarchakrapani** [reply]
Thanks for your suggestion I will try that please let me know the fork incase if you find it.
